alerts/cluster-monitoring-operator/NodeClockNotSynchronising.md

```diff
@@ -0,0 +1,50 @@
+# NodeClockNotSynchronising
+
+## Meaning
+
+This alert signifies issues with the NTP server on a node (mentioned in the alert).
+When the affected node is a master, this specifically becomes an issue
+as when the certs are rotated for the apiserver on that node, they fail
+validation due to the invalid time on that master.
+
+
+## Impact
+This alert is critical.
+It can lead to the apiserver operator going down and
+affecting other operators (like monitoring etc) after
+an extended period of apiserver operator being degraded.
+
+## Diagnosis
+
+1. Start a debug pod on the affected node and check the chronyd service.
+Start it if it's failing or stopped.
+
+```shell
+oc -n default debug node/<affected_node_name>
+systemctl status chronyd
+systemctl start chonyd
```

Comment:
Are we looking for any particular logs in the `status` messages or should the user run `journalctl -xe -u chronyd.service` ? 

`start` and `restart` makes more sense in the `# Mitigation` section.